.github/workflows/destroy_aks.yml

```diff
@@ -0,0 +1,46 @@
+name: Destroy Review Instance
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  destroy:
+    name: Destroy
+    environment:
+       name: review_aks
+    runs-on: ubuntu-latest
+    concurrency: review_aks${{github.event.number}}
```

Comment:
The string has to match exactly the concurrency string in the deployment to prevent them from running at the same time: https://github.com/DFE-Digital/schools-experience/blob/master/.github/workflows/build.yml#L539